### DISCUSSION TITLE
Recording videos and FTP:  starter guide?

### DISCUSSION THREAD

**Discussion by calcioscacchi** [roleoroleo/yi-hack-Allwinner-v2#983] [Q&A]
Hi there, 

I enjoy this firmware making my HA experience way better!

I would like to save my camera recordings continuously, deleting old files as new ones are created, and if possible, specifying the desired duration of the videos (for example, a video every half hour)

Also, I'd like to access to the files via FTP using FileZilla.
 
Can you please exlpain how to do it, expeccially the "FTP Options" configuration?

Thanks

**Comment by roleoroleo** [comment]
If "Disable cloud" is enable be sure to enable "Recording without cloud".
![immagine](https://github.com/user-attachments/assets/c7ed3d93-fea8-4d98-9644-2ef89c8b4a98)

About ftp, enable "FTP upload"
![immagine](https://github.com/user-attachments/assets/83cf39e6-8ee7-4db6-a9e4-78d54e720eb9)
and enable "FTP delete" and "Create directory tree" if you want.
Then you need to set host, base directory, username and password.
![immagine](https://github.com/user-attachments/assets/7db9d9b4-5965-45b3-868e-4964ac9078ce)

**Reply by calcioscacchi** [reply]
Thanks for your fast reply.

what if I enable the cloud?

second question, sounds stupid but can I set the ftp ip host with the ip of the camera or there is some service to link to?

**Comment by calcioscacchi** [comment]
I setted as you suggested but I can't access.... what do you use to access via FTP?
![image](https://github.com/user-attachments/assets/65fe8f88-23e7-42fc-9216-55cff513e42c)

**Comment by calcioscacchi** [comment]
I checked my ssh (with PuTTy) and ftp connection and in both cases the answer when I put the password is "Access denied" or "Login authentication failed".

Seems that it can't recognize any password, i tried to set an ampty password and still get the error.
